### DISCUSSION TITLE
Ghost container takes global domain

### DISCUSSION THREAD

**Discussion by JanSeW** [dokku/dokku#4623] [Q&A]
Hey guys,
Yesterday I started to play around with Dokku and besides deployment with `git push` I also tried deploying a docker image. For testing purposes I choose Ghost and found [this instructions](https://davidmargol.in/how-to-deploy-ghost-with-dokku/). I followed them except I used `git:from-image` instead of `tags:deploy` (and all the other deprecated commands as in Step 5 + 6). I also didn't created a lets encrypt cert because I do this for testing Dokku. I also added an external domain.
Ghost runs fine but it seems like it messes with the dokku proxy. After deployment it isn't reachable with the subdomain provided by dokku (vhost). Strangely the ghost site opens when I try to reach the global domain of the dokku server or the external domain. On the other side none of my other applications are reachable (i.e. Ghost is reachable on cool.domain.com but a NodeJS app on node.cool.domain.com isn't).
The server runs on `Debian 4.19.181` with `Dokku version 0.24.8`.
I'm not really into configuring Nginx but maybe some of you guys can help me with this problem.
Thank you!

**Comment by josegonzalez** [comment]
Do you have an updated link to the instructions? That link no longer works.

**Reply by JanSeW** [reply]
You can find a mostly similar instruction [here](https://matthisk.com/running-ghost-publishing-on-dokku/). I discovered that this behavior don't appear on Ubuntu, only on Debian. So my fix for now is to switch from Debian to Ubuntu.
